webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/SystemApi.scala

```diff
@@ -685,7 +687,9 @@ class SystemApiService11(
       getActor(req),
       Some("Technique library reloaded from REST API")
     ) match {
-      case Full(x) => Right(JField("techniques", "Started"))
+      case Full(_) =>
+        compilationStatusService.getUpdateAndSync().runNow
```

Comment:
It also certainly means that we didn't scope it right the last time. Now, the cache is not opaque anymore, and need to be managed both by `TechniqueWriterImpl` and the UpdateTechniqueLibrary, which is not good. It leads to think that these two services are not uncoupled in reality.

Reply:
ok, no we already did have that problem for technique deletion, and it's the reason why we have a `DeleteEditorTechnique` service. It looks like the `unsync` should be in `TechniqueArchiver` (because it looks like the only common point).

Reply:
Tinking more about it, it can go in `DeleteEditorTechnique` because: 
- error can only be induced by yaml technique compilation, so we only need to remove error for these ones (no error for other techniques can happen), 
- no historical technique from std lib are compiled - only the ones from editor are, 
- the `DeleteEditorTechnique` is about deleting editor technique, so the one producing errors.

Reply:
:white_check_mark: https://github.com/Normation/rudder/pull/6068/commits/7dad047401ff4cf54ec18b572aa06487aa9e718b